### PR TITLE
Improve Gradle snippet (Readme)

### PR DESCRIPTION
**Pull request by Animeshz**

### CODE REVIEW

**Comment by akuleshov7** [comment]
thank you for it! 😄

**Review by akuleshov7 — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -164,51 +164,54 @@ Add this plugin to your pom.xml:
...
 Add the code below to your `build.gradle.kts`:
 <details>
-  <summary><b>Gradle plugin snippet</b></summary><br>
+  <summary><b>Gradle snippet</b></summary><br>
   
 ```kotlin
-val ktlint by configurations.creating
+object Versions {
+    const val ktlint = "0.39.0"
+    const val diktat = "0.1.2"
```
Comment (petertrr) [inline README.md:177]:
```suggestion
    const val diktat = "0.1.3"
```
Our current version is already 0.1.3 :)

**Review by petertrr — CHANGES_REQUESTED** [review]

**Review by petertrr — APPROVED** [review]

**Comment by Animeshz** [comment]
@akuleshov7 Sorry to inform late, I have done a small mistake in the last task, i.e. in `diktatFormat`, the args should have -F option passed in order to auto-fix the files😅 -> `listOf("-F", ...)`. Since the PR is closed, I can't fix it, hopefully you can fix it.

**Pull request merged**